### PR TITLE
en: add error-handling.md

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

- merge `error-system.md` and `troubleshoot-dm.md` into `error-handling.md`
- update content in `error-handling.md`
- update TOC in https://github.com/pingcap/docs-dm/pull/133
- update table width (refer to https://github.com/pingcap/docs-dm/pull/136)

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/78
- Other reference link(s):

### CODE REVIEW

File: en/error-handling.md [hunk 1]
```diff
@@ -1,17 +1,96 @@
 ---
-title: Data Migration Error Handling
-summary: Learn how to deal with errors when you use TiDB Data Migration.
+title: Handle Errors
+summary: Learn about the error system and how to handle common errors when you use DM.
 category: reference
-aliases: ['/docs/tidb-data-migration/dev/error-handling/']
+aliases: ['/docs/tidb-data-migration/dev/error-handling/','/docs/tidb-data-migration/dev/troubleshoot-dm/','/docs/tidb-data-migration/dev/error-system/']
```
Comment (ran-huang) [inline en/error-handling.md:5]: Shall we add aliases in the new website style? Like `/tidb-data-migration/dev/error-system/`

File: en/error-handling.md [hunk 2]
```diff
@@ -1,17 +1,96 @@
 ---
-title: Data Migration Error Handling
-summary: Learn how to deal with errors when you use TiDB Data Migration.
+title: Handle Errors
+summary: Learn about the error system and how to handle common errors when you use DM.
 category: reference
```
Comment (ran-huang) [inline en/error-handling.md:4]:
```suggestion
category: troubleshoot
```

File: en/error-handling.md [hunk 3]
```diff
@@ -1,17 +1,96 @@
...
 
-## Common error description and handling method
+## Error system
 
-| <div style="width: 100px;">Error code</div>       | Error description                                                     |  Handling method                                                    |
+In the error system, usually, the information of a specific error is as follows:
+
+- `code`: error code.
+
+    DM uses the same error code for the same error type. An error code does not change as DM version changes.
```
Comment (ran-huang) [inline en/error-handling.md:18]:
```suggestion
    DM uses the same error code for the same error type. An error code does not change as the DM version changes.
```

File: en/error-handling.md (hunk 2 above)
Comment (toutdesuite) [reply en/error-handling.md:4]: Searched some docs. The category is either `how-to` or `ref`. Not sure whether I need to change the category to `troubleshoot`, though this does make sense

File: en/error-handling.md (hunk 1 above)
Comment (toutdesuite) [reply en/error-handling.md]: Updated. PTAL again, thanks!

File: en/error-handling.md (hunk 2 above)
Comment (ran-huang) [reply en/error-handling.md:4]: Let's discuss this issue with others in #sig-docs. I've been wondering how to write the "category" metadata for some time...🤔

Comment (toutdesuite) [reply en/error-handling.md]: updated in 9a85baddd7962c83cdb92a089ed93bf524bbb0ac~

**Pull request merged**